libpay-braintree/src/test/scala/com/wix/pay/braintree/it/BraintreeGatewayWithOAuthIT.scala

```diff
@@ -0,0 +1,46 @@
+package com.wix.pay.braintree.it
+
+import com.braintreegateway.Environment
+import com.wix.pay.PaymentGateway
+import com.wix.pay.braintree._
+import com.wix.pay.creditcard.{CreditCard, YearMonth}
+import com.wix.pay.model.CurrencyAmount
+import org.specs2.mutable.SpecWithJUnit
+
+
+class BraintreeGatewayWithOAuthIT extends SpecWithJUnit {
```

Comment:
ok will use wire mock (in serious TDD you test against Sandbox minimal set, and extended scenarios with a driver. if its stable you win it all, if its not you move just to the driver)